LEAF_Nexus/scripts/refreshOrgchartEmployees.php

```diff
@@ -156,7 +156,24 @@ function updateLocalOrgchartBatch()
         updateEmployeeDataBatch($localEmployeeUsernames);
     }
 }
+/**
+ * @param getEmployeeUIDs array $localEmployeeUsernames
+ * @return [$localEmpUIDs,$localEmpArray]
+ */
+function getEmployeeUIDs(array $localEmployeeUsernames): array {
+    global $oc_db;
 
+    // get local empuids, need to gather employees that have been added. 
+    $localEmployeeSql = "SELECT empUID, userName FROM employee WHERE userName IN (".implode(",",array_fill(1, count($localEmployeeUsernames), '?')).")";
```

Comment:
This needs to be split into the normal way of setting up sql queries.  We definitely don't want complex actions inside an assignment like this.

Reply:
Broken this out, there were a few other like this.